### PR TITLE
feat: add version in CLI

### PR DESCRIPTION
**Pull request by adriantam**
## Description
Allow printing out of version in CLI

```shell
adriantam@C02F5002MD6P openfga % ./openfga
A high performance and flexible authorization/permission engine built for developers and inspired by Google Zanzibar.

OpenFGA is designed to make it easy for developers to model their application permissions, and to add and integrate fine-grained authorization into their applications.
Complete documentation is available at https://openfga.dev.

Usage:
  openfga [command]

Available Commands:
  completion  Generate the autocompletion script for the specified shell
  help        Help about any command
  migrate     Run database schema migrations needed for the OpenFGA server
  run         Run the OpenFGA server
  version     Return the OpenFGA version

Flags:
  -h, --help   help for openfga

Use "openfga [command] --help" for more information about a command.
adriantam@C02F5002MD6P openfga % ./openfga  version
2023/03/10 15:02:35 OpenFGA Version dev
```

## References
N/A

## Review Checklist
- [x] I have clicked on ["allow edits by maintainers"](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/working-with-forks/allowing-changes-to-a-pull-request-branch-created-from-a-fork).
- [ ] I have added documentation for new/changed functionality in this PR or in a PR to [openfga.dev](https://github.com/openfga/openfga.dev) [Provide a link to any relevant PRs in the references section above]
- [x] The correct base branch is being used, if not `main`
- [ ] I have added tests to validate that the change in functionality is working as expected

### CODE REVIEW

**Review by miparnisari — APPROVED** [review]

File: cmd/version.go [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+		RunE:  version,
+		Args:  cobra.NoArgs,
+	}
+
+	return cmd
+}
+
+// print out the built version
+func version(_ *cobra.Command, _ []string) error {
+	log.Println("OpenFGA Version", build.Version)
```
Comment (jon-whit) [inline cmd/version.go:25]: Can you please add the other build information as well so we have the `commit` and `build date` etc..?

**Review by jon-whit — CHANGES_REQUESTED** [review]
Left one minor comment

File: cmd/version.go (hunk 1 above)
Comment (adriantam) [reply cmd/version.go]: Done

File: cmd/version.go [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+		RunE:  version,
+		Args:  cobra.NoArgs,
+	}
+
+	return cmd
+}
+
+// print out the built version
+func version(_ *cobra.Command, _ []string) error {
+	log.Printf("OpenFGA Version %s Date %s commit id %s ", build.Version, build.Date, build.Commit)
```
Comment (jon-whit) [inline cmd/version.go:25]: Minor nit pick, but do you mind if we format this like

> OpenFGA version 'v0.3.6' built from '3e8facb' on '2023-02-19T20:54:52Z'

Comment (adriantam) [reply cmd/version.go]: done

**Review by jon-whit — APPROVED** [review]

**Pull request merged**